docs/getStarted.md

```diff
@@ -284,7 +284,7 @@ When using "Student view without access restrictions", you have access to all as
 
 Learn more:
 
-- [Quick reference guide about question structure and PrairieLearn elements](https://coatless.github.io/pl-cheatsheets/pdfs/prairielearn-authoring-cheatsheet.pdf)
+- [Quick reference guide about question structure and PrairieLearn elements](https://web.archive.org/web/20230630091003/https://coatless.github.io/pl-cheatsheets/pdfs/prairielearn-authoring-cheatsheet.pdf)
```

Comment:
I'd advocate for just removing this. We don't control it and it will never be updated. If we like the contents, let's pull it into our own docs somehow.